tests/CommonTest.qs

```diff
@@ -52,8 +53,8 @@ namespace Tests {
     }
 
     // QRAM where only the last memory cell contains a 1
-    internal function LastCellFullQRAM(addressSize : Int) : MemoryBank {
-        return GeneratedMemoryBank([MemoryCell(2^addressSize - 1, [true])]);
+    internal function LastCellFullQRAM(addressBits : Int) : MemoryBank {
```

Comment:
AddressBits imply a bit that this is a mask or a value mask. Not that problematic, but given the rest uses the size word for this, might be better to keep it consistent in the file (unless you relaly mean a mask here).

Reply:
Yup, that should be addressSize in line with the rest.